File: test/fixtures/java-unirest/get_with_header_without_value.java
import kong.unirest.HttpResponse;
import kong.unirest.Unirest;

class Main {
    public static void main(String[] args) {
            HttpResponse<String> httpResponse
               = Unirest.get("http://localhost:28139/get")
            .header("Content-Type", "text/xml;charset=UTF-8")
            .header("getWorkOrderCancel", "")
            .asString();
    }
}

File: test/fixtures/kotlin-ktor/get_with_header_without_value.kt
import io.ktor.client.HttpClient
import io.ktor.client.request.get
import io.ktor.client.request.header
import io.ktor.client.statement.bodyAsText
import io.ktor.http.isSuccess
import kotlinx.coroutines.runBlocking

val client = HttpClient()

runBlocking {
    val response = client.get("http://localhost:28139/get") {
        header("Content-Type", "text/xml;charset=UTF-8")
        header("getWorkOrderCancel", "")
    }
    if (!response.status.isSuccess()) throw Exception("Request failed with status code ${response.status}")

    println(response.bodyAsText())
}
